core/src/main/java/jenkins/util/SystemProperties.java

```diff
@@ -60,40 +74,93 @@
  *
  * <p>While it looks like it on first glance, this cannot be mapped to {@link EnvVars},
  * because {@link EnvVars} is only for build variables, not Jenkins itself variables.
- *
- * @author Johannes Ernst
- * @since 2.4
  */
-//TODO: Define a correct design of this engine later. Should be accessible in libs (remoting, stapler) and Jenkins modules too
+@SuppressFBWarnings(value = "ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD", justification = "Currently Jenkins instance may have one ond only one context")
```

Comment:
typo?

Reply:
I think just @oleg-nenashev’s grammar, moved as is from line 85.